core/src/test/java/com/datastrato/gravitino/storage/TestEntityStorage.java

```diff
@@ -110,7 +110,7 @@ private void init(String type, Config config) {
 
   private void prepareJdbcTable() {
     // Read the ddl sql to create table
-    String scriptPath = "h2/h2-init.sql";
+    String scriptPath = "relational-entity-store/jdbc/h2/schema-0.5.0-h2.sql";
```

Comment:
What's the meaning of `0.5.0`?

Reply:
This refers to the next release version. We may add some ddl statements to the new version in the future. Users can execute initialization statements according to the access version. Referring to the implementation in Hive Metastore:
![image](https://github.com/datastrato/gravitino/assets/26177232/8c0bc384-03df-4bf4-b549-e7cd2d99658d)

Reply:
Got it.